x-pack/plugins/infra/server/lib/alerting/log_threshold/log_threshold_executor.ts

```diff
@@ -101,6 +177,44 @@ const getESQuery = (
   // Negative assertions (things that "must not" match)
   const mustNotFilters = buildFiltersForCriteria(negativeCriteria);
 
+  const rangeFilters = [
+    {
+      range: {
+        [sourceConfiguration.fields.timestamp]: {
+          gte: from,
+          lte: to,
+          format: 'epoch_millis',
+        },
+      },
+    },
+  ];
+
+  const sharedSubLevelAggregations = {
+    interval_buckets: {
+      date_histogram: {
```

Comment:
I wonder why we need a date histogram here. Is this even used anywhere? Shouldn't the range filter already select the one date interval that the alert checks?